src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -2306,12 +2306,10 @@ sealed interface Scope permits MemorySessionImpl {
         boolean isAlive();
 
         /**
-         * Returns {@code true}, if the provided object is also a scope, which models the same lifetime as that
-         * modelled by this scope. In that case, it is always the case that
+         * {@return {@code true}, if the provided object is also a scope, which models the same lifetime as that
+         * modelled by this scope}. In that case, it is always the case that
```

Comment:
This brace (after `scope`) seems spurious?

Reply:
It's to close the text span for the `{@return`

Reply:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/7457876/228246205-ff2730cb-610f-4673-aa30-d110845a34fc.png">

It belongs to the opening brace for `{@return`. Hard to see with other curls in between.

Reply:
I would have written:

```
{@return if the provided object ...
```

`true` seams redundant to me but I know this style is used within the JDK.

Reply:
Whoops - thanks!